### PR TITLE
search box issue resolved

### LINKED ISSUE #510: Account issue
- [ ] programme_curriculum/curriculums/2/

  issue : Search box at unnecessary modules with no change in view  & obsolete curriculum is not updated(blank)
 

- [ ] programme_curriculum/curriculum_semesters/9/

  issue : phd curriculum is empty otherwise this sub link could be eliminated from the module(if phd has no curriculum)
 

- [ ] programme_curriculum/batches/

issue : Finished batches are not visible

@tushhr

### PR DESCRIPTION
**Pull request by Shrestha25**
Search box at unnecessary modules with no change in view & obsolete curriculum is not updated(blank)

### CODE REVIEW

**Comment by Shrestha25** [comment]
review @tushhr

File: FusionIIIT/templates/programme_curriculum/view_curriculums_of_a_programme.html [hunk 1]
```diff
@@ -139,7 +139,7 @@
 {% block rightcontent %}
 <br>
 <br>
-<div class="ui center aligned segment">
+<!-- <div class="ui center aligned segment">
```
Comment (tushhr) [inline FusionIIIT/templates/programme_curriculum/view_curriculums_of_a_programme.html]: Don't comment it, just Remove the whole part..

Comment (Shrestha25) [reply FusionIIIT/templates/programme_curriculum/view_curriculums_of_a_programme.html]: Done .

**Comment by MananJethwani** [comment]
@tushhr kindly review if the issue is resolved.
thanks.

**Comment by tushhr** [comment]
Yup, perfect.

Can you merge it @MananJethwani?

**Pull request merged**